code/modules/mining/abandoned_crates.dm

```diff
@@ -11,6 +11,7 @@
 	var/codelen = 4
 	var/qdel_on_open = FALSE
 	var/spawned_loot = FALSE
+	var/loot
```

Comment:
I don't like this. it doesn't provide context on what is spawned. It's a half step that could just be storing the typepaths to spawn, but typepaths require extra context that you can't store here.
Is this just for debugging?

Reply:
I would like that yeah